examples/server/x11.c

```diff
@@ -136,8 +137,8 @@ void get_window_size(xcb_connection_t* conn, xcb_window_t window, uint16_t* widt
 
 void get_window_image(xcb_connection_t* conn, xcb_window_t window, uint8_t* buff)
 {
-    uint16_t width = 0;
-    uint16_t height = 0;
+    int16_t width = 0;
```

Comment:
Type change needed here?

Reply:
Also not needed, I think I need more coffee